codes/classical/q-ary_digits/rank_modulation.yml

```diff
@@ -0,0 +1,43 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+# code id, physical, logical are all lower case
+code_id: rank_modulation
+physical: groups
+logical: q-ary_digits
```

Comment:
not clear what is "groups". The best is to say that it encodes a set of M messages into permutations on n elements. Therefore, "logical" shouldn't be q-ary digits because q is not defined.

Reply:
physical: groups
logical: points